builtin/providers/heroku/resource_heroku_space_test.go

```diff
@@ -0,0 +1,114 @@
+package heroku
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	heroku "github.com/cyberdelia/heroku-go/v3"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccHerokuSpace_Basic(t *testing.T) {
+	var space heroku.SpaceInfoResult
+	spaceName := fmt.Sprintf("tftest-%s", acctest.RandString(10))
+	spaceName2 := fmt.Sprintf("tftest-%s", acctest.RandString(10))
+	org := os.Getenv("HEROKU_ORGANIZATION")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			if org == "" {
+				t.Skip("HEROKU_ORGANIZATION is not set; skipping test.")
+			}
+		},
```

Comment:
Okay cool, if the error returned from the API is sufficiently descriptive enough, I don't think we'll need to add the env-var.